dtfs-central-api/api-tests/v1/utilisation-reports/put-keying-data-mark-as-done.api-test.ts

```diff
@@ -106,11 +112,16 @@ describe(`PUT ${BASE_URL}`, () => {
     expect(status).toBe(200);
   });
 
-  it('sets fee record status to RECONCILED', async () => {
+  it('sets fee record status to RECONCILED, reconciledByUserId to the user who performed the action and the dateReconciled to now', async () => {
     // Arrange
     const reportId = 1;
     const report = UtilisationReportEntityMockBuilder.forStatus('RECONCILIATION_IN_PROGRESS').withId(reportId).build();
-    const feeRecord = FeeRecordEntityMockBuilder.forReport(report).withId(1).withStatus('READY_TO_KEY').build();
+    const feeRecord = FeeRecordEntityMockBuilder.forReport(report)
+      .withId(1)
+      .withStatus(FEE_RECORD_STATUS.READY_TO_KEY)
+      .withReconciledByUserId(null)
+      .withDateReconciled(null)
```

Comment:
The mock builder does default these values to `null` by default - declaring them in this test just makes it explicit to the reader that these values are starting as `null` (and also future proofs it in case someone updates the mock builder to default either of these fields to non-`null` values)